src/components/search.js

```diff
@@ -28,18 +89,12 @@ export default class Search extends Component {
       <div className='search-wrapper'>
         <Breadcrumbs title={config.page.search.title} color={config.page.search.color} bgcolor={config.page.search.bgcolor} />
         <div className='search-content block-content'>
+          <div className='searchbox'>
```

Comment:
@phuongdk Break this div into new components (SearchBox/SearchInput/SearchResult...).